backend/app/exporters/serializers/mods.rb

```diff
@@ -143,4 +161,49 @@ def serialize_note(note, xml)
       xml.text note.content
     }
   end
+
+  private
+
+  def process_date_string(date)
+    # use date expression, if present
+    unless date['expression'].nil? || date['expression'].empty?
+      return date['expression']
+    end
+
+    # if end date specified, use begin - end form
+    unless date['end'].nil? || date['end'].empty?
+      return "#{date['begin']} - #{date['end']}"
+
+    # otherwise, just use begin date
```

Comment:
@avatar382 @cdibella Referring to lines 39-60 in the spreadsheet, if you have a date['begin'] and a date['end'], shouldn't you have separate date tags for each along with the date['expression'] tag?